lib/question.js

```diff
@@ -10,8 +10,9 @@ const moment = require('moment');
 const util = require('util');
 const fs = require('fs');
 const unzipper = require('unzipper');
-const showdown = require('showdown');
+const xss = require('xss');
```

Comment:
Could we have remark/rehype filter out anything to do with scripting instead? IIRC you can configure it to do so, though I don't know exactly how off the top of my head.

Reply:
Yup: https://github.com/rehypejs/rehype-sanitize. I'll check it out.